<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>苹果HEIC转JPG转换器 v0.1.1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="titleBar" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#titleBar {
    background-color: white;
    border-bottom: 1px solid #DDDDDD;
}</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="titleLabel">
         <property name="styleSheet">
          <string notr="true">QLabel {
    color: #333333;
    font-size: 16px;
    font-weight: bold;
    font-family: &quot;Microsoft YaHei&quot;;
}</string>
         </property>
         <property name="text">
          <string>苹果HEIC转JPG转换器 v0.1.1</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="loginButton">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border: none;
    color: #666666;
    background: transparent;
    font-size: 16px;
    font-weight: bold;
    font-family: &quot;Microsoft YaHei&quot;;
}
QPushButton:hover {
    color: #FF8533;
}</string>
         </property>
         <property name="text">
          <string>未登录</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="serviceButton">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border: none;
    color: #666666;
    background: transparent;
    font-size: 16px;
    font-weight: bold;
    font-family: &quot;Microsoft YaHei&quot;;
}
QPushButton:hover {
    color: #FF8533;
}</string>
         </property>
         <property name="text">
          <string>客服咨询</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="minimizeButton">
         <property name="minimumSize">
          <size>
           <width>45</width>
           <height>30</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border: none;
    background: transparent;
}
QPushButton:hover {
    background-color: #E5E5E5;
}
QPushButton:pressed {
    background-color: #D5D5D5;
}</string>
         </property>
         <property name="text">
          <string>一</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="maximizeButton">
         <property name="minimumSize">
          <size>
           <width>45</width>
           <height>30</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border: none;
    background: transparent;
}
QPushButton:hover {
    background-color: #E5E5E5;
}
QPushButton:pressed {
    background-color: #D5D5D5;
}</string>
         </property>
         <property name="text">
          <string>口</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="closeButton">
         <property name="minimumSize">
          <size>
           <width>45</width>
           <height>30</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border: none;
    background: transparent;
}
QPushButton:hover {
    background-color: #FF0000;
    color: white;
}
QPushButton:pressed {
    background-color: #AA0000;
    color: white;
}</string>
         </property>
         <property name="text">
          <string>×</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="contentWidget" native="true">
      <layout class="QVBoxLayout" name="contentLayout">
       <item>
        <widget class="QWidget" name="topWidget" native="true">
         <layout class="QHBoxLayout" name="topLayout">
          <item>
           <widget class="QPushButton" name="addFileButton">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>32</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: #FF8533;
    color: white;
    border: none;
    border-radius: 4px;
    padding: 5px 15px;
    font-size: 14px;
}
QPushButton:hover {
    background-color: #FF9955;
}
QPushButton:pressed {
    background-color: #FF7722;
}</string>
            </property>
            <property name="text">
             <string>添加文件</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="addFolderButton">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>32</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: #FF8533;
    color: white;
    border: none;
    border-radius: 4px;
    padding: 5px 15px;
    font-size: 14px;
}
QPushButton:hover {
    background-color: #FF9955;
}
QPushButton:pressed {
    background-color: #FF7722;
}</string>
            </property>
            <property name="text">
             <string>添加文件夹</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="clearButton">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>32</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: #FF8533;
    color: white;
    border: none;
    border-radius: 4px;
    padding: 5px 15px;
    font-size: 14px;
}
QPushButton:hover {
    background-color: #FF9955;
}
QPushButton:pressed {
    background-color: #FF7722;
}</string>
            </property>
            <property name="text">
             <string>清除列表</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="startConvertButton">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>32</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: #FF8533;
    color: white;
    border: none;
    border-radius: 4px;
    padding: 5px 15px;
    font-size: 14px;
}
QPushButton:hover {
    background-color: #FF9955;
}
QPushButton:pressed {
    background-color: #FF7722;
}
QPushButton:disabled {
    background-color: #CCCCCC;
}</string>
            </property>
            <property name="text">
             <string>开始转换</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="dropPage">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="dropLabel">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>200</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel {
    border: 2px dashed #CCCCCC;
    border-radius: 10px;
    color: #666666;
    font-size: 16px;
}</string>
             </property>
             <property name="text">
              <string>点击添加图片或拖拽图片到此处</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="listPage">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QListWidget" name="fileListWidget"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
